qiita_db/job.py

```diff
@@ -92,6 +93,10 @@ def exists(cls, datatype, command, options, analysis,
             Options for the command in the format {option: value}
         analysis : Analysis object
             The analysis the job will be attached to on creation
+        input_file_reference_id : int
```

Comment:
Normally these are the string versions, with ID conversion called in the function (see datatype and command in this function). Why passing the ID directly here?